webrtc.html

```diff
@@ -7318,13 +7317,14 @@ <h2>Dictionary RTCRtpContributingSource Members</h2>
           <dl data-link-for="RTCRtpContributingSource" data-dfn-for=
           "RTCRtpContributingSource" class="dictionary-members">
             <dt data-tests="RTCRtpReceiver-getSynchronizationSources.https.html"><dfn data-idl><code>timestamp</code></dfn> of type <span class=
-            "idlMemberType">DOMHighResTimeStamp</span>, required</dt>
+            "idlMemberType"><a data-cite="!HIGHRES-TIME#dom-domhighrestimestamp">DOMHighResTimeStamp</a></span>, required</dt>
             <dd>
-              <p>The timestamp of type DOMHighResTimeStamp [[!HIGHRES-TIME]],
-              indicating the most recent time of playout of media that arrived
-              in an RTP packet originating from this source. The timestamp is
-              defined as <a>performance.timeOrigin</a> +
-              <a>performance.now()</a> at the time of playout.</p>
+              <p>The <code>timestamp</code> indicating the most recent time a
+              frame from an RTP packet, originating from this source, was
+              delivered to the <code><a>RTCRtpReceiver</a></code>'s
+              <code><a>MediaStreamTrack</a></code>. The <code>timestamp</code>
+              is defined as <a>performance.timeOrigin</a> +
+              <a>performance.now()</a> at the time of delivery.</p>
```

Comment:
Here you want to change the timestamp from "playout time" to "delivery time". What does "delivery" mean, and when does it happen?
(In particular, whether it happens before or after the jitter buffer.)

Reply:
"Delivered" is defined in the immediately preceding sentence. Would it make sense to just change the text from "at the time of delivery" to "at that time"?

The instant it's referring to is when a frame is passed from `RTCRtpReceiver` to `MediaStreamTrack`. That would be after the jitter buffer, decoder, sync buffer, and all the other WebRTC bits.